test-e2e/k8spod.py

```diff
@@ -14,3 +15,76 @@ def ping(self, ip):
             return False
         return True
 
+    def http(self, client_name):
+        httpd_cmd = '/bin/sh -c python3 -m http.server'
+        self.api.pod_exec2(self.name, httpd_cmd)
+        sleep(1)
+        #print("OUTPUT {}".format(err))
+
+        curl_cmd = 'curl http://{}:8000 -Ss -m 1'.format(self.ip)
+        self.api.pod_exec2(client_name, curl_cmd)
+#        err = self.api.pod_exec(client_name, curl_cmd)
+#        print("OUTPUT CURL {}".format(err))
+#        if err['status'] != "Success":
+#            return False
+        return True
+
+    def tcp(self, client_name):
+        output_dir = '/tmp'
+        tcp_recv_file = '{}/{}_{}_tcp_recv'.format(output_dir, self.name, self.ip)
+        tcp_sent_file = '{}/{}_{}_tcp_sent'.format(output_dir, self.name, self.ip)
+
+        cmd_nc = '/bin/sh -c (nc -l -p 9001 > {})'.format(tcp_recv_file)
+        err = self.api.pod_exec(self.name, cmd_nc)
+        print("OUTPUT NC {}".format(err))
+        if err['status'] != "Success":
+            return False
+
+        self.tcp_client(client_name, self.ip, tcp_sent_file)
+        
+        cmd_diff = 'diff {} {}'.format(tcp_recv_file, tcp_sent_file)
+        err = self.api.pod_exec(client_name, cmd_diff)
+        if err['status'] != "Success":
+            return False
+        return True
```

Comment:
The traffic verification should be in a different method preferably in a different folder like in ```https://github.com/futurewei-cloud/mizar/blob/dev-next/test/trn_func_tests/helper.py```